packages/user/src/components/UserMenu.tsx

```diff
@@ -1,6 +1,6 @@
 import { useContext } from "react";
 
-import { UserContextType } from "@/types";
+import { UserContextType } from "@/types/types";
```

Comment:
@shailesh896 `types/types`: can we have `index.ts` inside `types` and just import `@/types`